tensorflow/python/training/learning_rate_decay.py

```diff
@@ -84,3 +86,49 @@ def exponential_decay(learning_rate, global_step, decay_steps, decay_rate,
     if staircase:
       p = math_ops.floor(p)
     return math_ops.mul(learning_rate, math_ops.pow(decay_rate, p), name=name)
+
```

Comment:
Minor style nit: can you add another blank line here (our linter will complain otherwise...)?